wpilibj/src/main/java/edu/wpi/first/wpilibj/counter/Tachometer.java

```diff
@@ -114,7 +114,26 @@ public double getRevolutionsPerMinute() {
     if (edgesPerRevolution == 0) {
       return 0;
     }
-    return ((1.0 / edgesPerRevolution) / period) * 60;
+    return (1.0 / edgesPerRevolution) / period;
```

Comment:
This should still be minutes, right?

I'm guessing this doesn't just call getRevolutionsPerSecond for perf reasons?